package-lock.json

```diff
@@ -725,20 +725,13 @@
       "dev": true
     },
     "clipboard": {
-      "version": "1.7.1",
-      "resolved": "https://registry.npmjs.org/clipboard/-/clipboard-1.7.1.tgz",
-      "integrity": "sha1-Ng1taUbpmnof7zleQrqStem1oWs=",
+      "version": "1.6.1",
+      "resolved": "https://registry.npmjs.org/clipboard/-/clipboard-1.6.1.tgz",
+      "integrity": "sha1-ZcW2VIEkZrD6q4Lca6Dx0vjkvlM=",
```

Comment:
RE: `clipboard` -> Bumped version to `1.7.1`. This trickled down and updated `tiny-emitter`.

RE: `jasmine-core`: Oddly, `grunt-contrib-jasmine` depends on a previous version of `jasmine-core`.

Verified that the tests pass 👍